src/schema/reserve-company-name.ts

```diff
@@ -0,0 +1,262 @@
+import { barbadosParishes } from "@/data/constants";
+import type { FormStep } from "@/types";
+
+export const formSteps: FormStep[] = [
+  {
+    id: "what-will-this-name-be-used-for",
+    title: "What will this name be used for?",
+    description: "",
+    fields: [
+      {
+        name: "aNewCompany",
+        label: "What will this name be used for?",
+        type: "radio",
+        hidden: true,
+        validation: { required: "Name will be used for is required" },
+        options: [
+          {
+            label: "A new company",
+            value: "newCompany",
+          },
+          {
+            label: "A change of company name",
+            value: "nameChange",
+          },
+          {
+            label: "A merger of two or more companies (amalgamation)",
+            value: "merger",
+          },
+        ],
+      },
+    ],
+  },
+  {
+    id: "company-present-name",
+    title: "What is the present name of the company?",
+    conditionalOn: {
+      field: "aNewCompany",
+      value: "nameChange",
+    },
+    description: "",
+    fields: [
+      {
+        name: "What is the present name of the company",
```

Comment:
```suggestion
        name: "companyName",
```
We don't want to use spaces for the field names